charts/gardener-extension-admission-calico/charts/runtime/templates/vpa.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       minAllowed:
         cpu: {{ required ".Values.global.vpa.resourcePolicy.minAllowed.cpu is required" .Values.global.vpa.resourcePolicy.minAllowed.cpu }}
         memory: {{ required ".Values.global.vpa.resourcePolicy.minAllowed.memory is required" .Values.global.vpa.resourcePolicy.minAllowed.memory }}
+      maxAllowed:
+        cpu: {{ required ".Values.global.vpa.resourcePolicy.maxAllowed.cpu is required" .Values.global.vpa.resourcePolicy.maxAllowed.cpu }}
+        memory: {{ required ".Values.global.vpa.resourcePolicy.maxAllowed.memory is required" .Values.global.vpa.resourcePolicy.maxAllowed.memory }}
```

Comment:
Could you please add `if` blocks analogous to `charts/gardener-extension-networking-calico/templates/vpa.yaml#L12`?
I do not understand why the admission controller would require setting min/max while the extension controller does not.

Reply:
Done